test/_shared.js

```diff
@@ -0,0 +1,49 @@
+
+var fs = require('fs');
+
+var nearley = require('../lib/nearley.js');
+var Compile = require('../lib/compile.js');
+var parserGrammar = require('../lib/nearley-language-bootstrapped.js');
+var generate = require('../lib/generate.js');
+
+function parse(grammar, input) {
+    if (grammar.should) {
```

Comment:
I didn't actually write launch.js, so that's totally not on me! :P I think we should take it out, though.